apps/amo/tests/__init__.py

```diff
@@ -340,7 +340,7 @@ def assert3xx(self, response, expected_url, status_code=302,
                   target_status_code=200):
         """Asserts redirect and final redirect matches expected URL.
 
-        Similar to Django's `assertRedirects` but skips the final GET
+        Similar to Django's `assert3xx` but skips the final GET
         verification for speed.
 
         """
```

Comment:
I believe the whole code in there is a copy/paste from `django.test.testcases.SimpleTestCase.assertRedirects` (see https://github.com/django/django/blob/master/django/test/testcases.py#L246).

It seems that starting with [django starting with 1.7](https://github.com/django/django/blob/stable/1.7.x/django/test/testcases.py#L248) this helper has a `fetch_redirect_response` parameter (which defaults to `True`) which could be set to False. The only code needed (instead of the full copy/paste) would then be:
```python
def assertLoginRedirects(self, response, to, status_code=302):
    return self.assertRedirects(response, to, status_code=302, fetch_redirect_response=False)
```

However, this parameter doesn't exist in [django 1.6](https://github.com/django/django/blob/stable/1.6.x/django/test/testcases.py#L247). As @EnTeQuAk is working on switching to it, maybe we could add a new issue to do that when it's done?